xds/src/main/java/io/grpc/xds/XdsClientMetricReporterImpl.java

```diff
@@ -199,11 +216,11 @@ static final class MetricReporterCallback implements ServerConnectionCallback {
       this.target = target;
     }
 
-    // TODO(dnvindhya): include the "authority" label once xds.authority is available.
-    void reportResourceCountGauge(long resourceCount, String cacheState,
+    void reportResourceCountGauge(String authority, long resourceCount, String cacheState,
```

Comment:
Keep `resourceCount` at the beginning, so the labels are together? (authority seems to fit well between resourceCount and cacheState)